presto-orc/src/main/java/com/facebook/presto/orc/cache/OrcFileTailSource.java

```diff
@@ -23,6 +23,8 @@
 
 public interface OrcFileTailSource
 {
+    int EXPECTED_FOOTER_SIZE_BYTES = 16 * 1024;
```

Comment:
- nit: EXPECTED_FOOTER_SIZE_IN_BYTES or EXPECTED_FOOTER_BYTES
- we don't usually put consts in interfaces. Maybe `OrcDataSourceUtils` is a good home? Or some other util or config classes.